lib/inquisition/auditors/backend/rails_best_practices/formatter.rb

```diff
@@ -0,0 +1,39 @@
+module Inquisition
+  module Auditors
+    module Backend
+      module RailsBestPractices
+        class Formatter < Core::BaseFormatter
+          REPORT_KEYS = {
+            linterable_name: 'filename',
```

Comment:
```suggestion
            lintable_name: 'filename',
```

linterable is incorrect wording, there is no such word in english language :)